### PR TITLE
new package: orpie 1.5.2

### PR DESCRIPTION
**Pull request by obadz**

### CODE REVIEW

File: pkgs/applications/misc/orpie/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  src = fetchurl {
+    url = "http://pessimization.com/software/orpie/${name}.tar.gz";
+    sha256 = "0v9xgpcf186ni55rkmx008msyszw0ypd6rd98hgwpih8yv3pymfy";
+  };
+
+  buildInputs = [ ncurses gsl ] ++ (with ocamlPackages; [ ocaml ]);
+
+  meta = {
+    homepage = http://pessimization.com/software/orpie/;
+    description = "Orpie is a fullscreen RPN calculator for the console.";
```
Comment (bjornfor) [inline pkgs/applications/misc/orpie/default.nix]: Description should not include package name, nor end with a dot.

**Comment by bjornfor** [comment]
Commit message should be "orpie: init at 1.5.2" (according to CONTRIBUTING.md).

**Comment by obadz** [comment]
fixed

**Comment by bjornfor** [comment]
Thanks. Added to master (db31c1c4388a9e18665cd8df157fde7613043c32) and release-15.09 (47026669bacf2e7838b4fee8eca037e4d8e091e0).

I added meta.platforms, to build binaries on the build farm.

**Pull request closed without merging**